utils/parametric/_library_client.py

```diff
@@ -350,6 +353,23 @@ def http_client_request(self, method: str, url: str, headers: List[Tuple[str, st
         ).json()
         return resp
 
+    def get_tracer_config(self) -> dict:
```

Comment:
It would be nice if the type of the returned dictionary was well defined. Something like:

```suggestion
    def get_tracer_config(self) -> Dict[str, str]:
```

OR can `get_tracer_config` return an object that has well defined fields. This might be easier to maintain